dbms/src/Storages/transformQueryForExternalDatabase.cpp

```diff
@@ -141,16 +141,12 @@ String transformQueryForExternalDatabase(
             if (function->name == "and")
             {
                 bool compatible_found = false;
-                auto new_function_and = std::make_shared<ASTFunction>();
-                auto new_function_and_arguments = std::make_shared<ASTExpressionList>();
-                new_function_and->arguments = new_function_and_arguments;
-                new_function_and->children.push_back(new_function_and_arguments);
```

Comment:
This may be important (adding functions arguments as a child).

Reply:
Probably you are right.

Reply:
Actually `makeASTFunction()` does the same.